docs/en/revocation-lists.rst

```diff
@@ -456,43 +578,43 @@ Law-Enforcement Authorities or Third Parties authorized by national law, MAY req
 Credential Proof of Possession
 ------------------------------
 
-The Credential Proof of Possession (**credential_pop**) MUST be a JWT that MUST contain the parameters (JOSE Header and claims) in the following table.
+The Credential Proof of Possession (**credential_pop**) MUST be a JWT or a CWT that MUST contain the parameters (Header and Payload) in the following table.
 
 .. list-table:: 
     :widths: 20 60 20
     :header-rows: 1
 
-    * - **JOSE header**
+    * - **Header**
       - **Description**
       - **Reference**
     * - **typ**
-      - In case of revocation request it MUST be set to ``revocation-request+jwt``. In case of Status Attestation request it MUST be set to ``status-attestation-request+jwt``, according to `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-status-attestations/01/>`_.
+      - In case of revocation request it MUST be set to ``revocation-request+{jwt,cwt}``. In case of Status Assertion request it MUST be set to ``status-assertion-request+{jwt,cwt}``, according to `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-status-assertions/01/>`_.
```

Comment:
```suggestion
      - In case of revocation request it MUST be set to ``revocation-request+{jwt,cwt}``. In case of Status Assertion request it MUST be set to ``status-assertion-request+{jwt,cwt}``, according to `[OAuth Status Assertions] <https://datatracker.ietf.org/doc/draft-demarco-oauth-status-assertions>`_.
```

see my previous comment